WordPress/Docs/Arrays/ArrayIndentationStandard.xml

```diff
@@ -0,0 +1,52 @@
+<documentation title="Array Indentation">
+    <standard>
+    <![CDATA[
+      The array closing bracket and indentation should line up with the start of the content on the line containing the array opener.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Closing bracket lined up correctly">
+        <![CDATA[
+$args = array(
+    'post_id' <em>=></em> 22,
```

Comment:
The emphasis tags aren't used correctly here - they should be around the closing parenthesis, since that's the thing that changes between the valid and invalid versions.